src/core/Wizard/Wizard.tsx

```diff
@@ -108,17 +107,17 @@ export const Wizard = (props: WizardProps) => {
               description={s.description}
             />
           ))}
-        </div>
+        </ol>
         {type === 'long' && (
-          <p className='iui-wizards-steps'>
-            <span className='iui-wizards-steps-count'>
+          <div className='iui-wizard-steps-label'>
```

Comment:
Tests missing for these classes...

Reply:
Yeah... we were not testing these classes previously either

Reply:
Added